tmexp/io_constants.py

```diff
@@ -1,5 +1,62 @@
+from typing import Counter as CounterType, Dict, List, NamedTuple, Optional, Set
+
 DATASET_DIR = "/data/datasets"
 
+
+class FileInfo(NamedTuple):
+    blob_hash: str
+    language: str
+
+
+class FilesInfo(dict):
```

Comment:
Yeah I had had a bug when doing that initiallt because I was using `self.__dict__[something]` instead of `self[something]` and forgot to change back ^^"